content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -732,6 +732,7 @@ Default values can be:
 
 - Static values that correspond to the field type, such as `5` (`Int`), `Hello` (`String`), or `false` (`Boolean`)
 - [Functions](#using-functions), such as [`now()`](/reference/api-reference/prisma-schema-reference#now) or [`uuid()`](/reference/api-reference/prisma-schema-reference#uuid)
+- [Scalar lists](/reference/api-reference/prisma-schema-reference#-modifier) <span class="api"></span> in versions `4.0.0` and later, when using databases that support them natively (PostgreSQL, CockroachDB and MongoDB)
```

Comment:
Are scalar lists technically not also "static values" like in the first item of this list? Just fancy ones.

Reply:
Hopefully this is better now